x-pack/plugins/cases/public/components/all_cases/translations.ts

```diff
@@ -105,3 +105,12 @@ export const STATUS = i18n.translate('xpack.cases.caseTable.status', {
 export const CHANGE_STATUS = i18n.translate('xpack.cases.caseTable.changeStatus', {
   defaultMessage: 'Change status',
 });
+
+export const MTTR_STAT = i18n.translate('xpack.cases.casesStats.mttr', {
+  defaultMessage: 'MTTR',
+});
+
+export const MTTR_DESCRIPTION = i18n.translate('xpack.cases.casesStats.mttrDescription', {
+  defaultMessage:
+    'MTTR (mean time to respond) is the average time it takes to recover from a product or system failure from the time when you are first alerted to that failure. This does not include any lag time in your alert system. ',
```

Comment:
We should change this to something that makes sense for Cases. @lcawl @nastasha-solomon @paul-tavares Any ideas?

Reply:
🤔 wrong Paul? 😀

Reply:
🙂 yeap! Sorry for the noise!

Reply:
cc @paulewing

Reply:
Is this calculation the time taken between Case open to case close? 
In which case, surely this is mean time to resolve.

Reply:
Oh, you are right! Yes, it is the time taken from open to close. `closed_at - created_at` in seconds.

Reply:
Hey, thanks for the tag @cnasikas ! Will the the MTTR metric replace the **Duration from creation to close** metric in 8.3? If so, I think the summary for **Duration from creation to close** still works for MTTR:
`Time elapsed from when the case was created to when it was closed.`

If MTTR is being included _in addition to_ **Duration from creation to close** I'd need to know more about how the two are different before suggesting a description.

Reply:
Hey team, I also would recommend a more exact name than MTTR (then we may not have to define within the info icon). For instance, "Avg Time from Open to Close" or more simply "Avg Time to Close."

Reply:
I agree with @paulewing that it's clearer to use a title like "Average duration" or "Average time to close". Especially since MTTR seems to have lots of different meanings per https://en.wikipedia.org/wiki/MTTR (i.e. "repair", "recovery", "restore"). If we need to stick with that acronym, however, here's some tooltip text for consideration:

> Mean time to respond (MTTR) is the average duration of cases from creation to closure.

Reply:
I updated the UI to reflect the changes. (see the description images)